install_scripts/templates/kubernetes/node-join.sh

```diff
@@ -264,6 +267,11 @@ if [ "$AIRGAP" = "1" ]; then
         mkdir -p "/etc/docker/certs.d/$DAEMON_REGISTRY_ADDRESS"
         promptForCA
         echo "$(echo "$CA" | base64 --decode)" > "/etc/docker/certs.d/$DAEMON_REGISTRY_ADDRESS/ca.crt"
+
+        promptForCert
```

Comment:
is a cert always required? if not can you make a flag to skip this prompt and maybe even just make it an argument and not prompt? this is gonna mess with automation

Reply:
good catch.  I'll just remove the prompt